Unity/Champloo/Assets/Scripts/Player/Movement States/InAttack.cs

```diff
@@ -14,6 +16,13 @@ protected override void Start()
 
     public override Vector3 ApplyFriction(Vector3 velocity)
     {
+        if (playerWeapon.IsSpecialAttacking && playerWeapon.SpecialAttackState == TimingState.WARMUP)
+        {
+            return Vector3.Lerp(
+                velocity,
+                Vector3.zero,
+                slowdownLerp);
+        }
```

Comment:
Is there a value for slowdownLerp that will make there be no slowdown? Otherwise see this affecting things like Hitomi Sword throw

Reply:
Yep! If you set it to 1 it will just go straight to zero. If you want, I can move this to a child implementation of InAttack

Reply:
What I mean is that is there a way to set it so that slowdown and stoppage never occur? Darkblade is the only character who stops in their attack special. Might as well put this in a child for him.

Reply:
If you don't want the player to stop at all, then you can set slowdown lerp to zero. I'll move it to a child implementation.